torchdynamo/variables/torch.py

```diff
@@ -398,3 +401,115 @@ def handle_ntuple(value):
             return variables.LambdaVariable(handle_ntuple, **options)
         else:
             return handle_ntuple(args[0])
+
+
+class TorchPyOperator(VariableTracker):
+    def __init__(self, value, **kwargs):
+        super(TorchPyOperator, self).__init__(**kwargs)
+        self.value = value
+
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        from torch.utils._python_dispatch import TorchDispatchMode
+        from torch.utils._python_dispatch import enable_torch_dispatch_mode
+
+        from . import ListVariable
+        from . import TensorVariable
+        from . import UserFunctionVariable
+
+        assert kwargs is None or len(kwargs) == 0, "kwargs are not supported, yet"
+
+        def unwrap_real(arg):
+            if isinstance(arg, TensorVariable):
+                return arg.as_proxy().node.meta["example_value"]
+            if isinstance(arg, UserFunctionVariable):
+                return arg.fn
+            if arg.has_unpack_var_sequence(tx):
+                return [
+                    unwrap_real(arg_inner) for arg_inner in arg.unpack_var_sequence(tx)
+                ]
+            return arg
+
+        # Get values
+        u_args = [unwrap_real(arg) for arg in args]
+
+        # Make fx to get a graph out (All PyOperator's should produce valid fx)
+        graph = make_fx(self.value)(*u_args)
```

Comment:
Could we just inline the python function of the operator rather than tracing it?

Reply:
Done! good suggestion.